service/src/main/java/bio/terra/workspace/service/workspace/GcpCloudSyncRoleMapping.java

```diff
@@ -141,15 +140,8 @@ public class GcpCloudSyncRoleMapping {
           "monitoring.metricDescriptors.list",
           "monitoring.monitoredResourceDescriptors.get",
           "monitoring.monitoredResourceDescriptors.list");
-
   private final List<String> additionalDataprocOwnerPermissions =
       ImmutableList.of(
-          "dataproc.clusters.create",
```

Comment:
i hope the iam sync backfill endpoint don't remove permissions

Reply:
Oh good point.. that would break existing workspaces with dataproc clusters set up if someone decides to run the backfill. The good news is that once the dataproc cluster work land, users can create and manager clusters in the UI in those projects. Do we also sync new resource policies?

Reply:
ah ok that's fine right since the resource is newly created so we don't need backfill on those.

Reply:
But I believe the resource policies are created on workspace creation right? The iam cloud sync step will fail to find those policies if they don't exist on older projects right? 🤔

Reply:
ah i just checked that it does add the resource custom roles: https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/main/java/bio/terra/workspace/service/admin/flights/cloudcontexts/gcp/GcpIamCustomRolePatchStep.java#L49C63-L49C63